.ci/jenkins/Jenkinsfile

```diff
@@ -76,16 +85,19 @@ def RunToxTests(configurationMaps) {
     UTILS.CreateStage("Run_tox_tests", runToxTestsMethod, configurationMaps)
 }
 
-def Cleanup(configurationMaps) {
-    Closure cleanupMethod = { configMap ->
+def Cleanup() {
+    stage("Cleanup"){
```

Comment:
Why do you remove closures? Currently you don't have configMap variable so this method will fail

Reply:
Curiously enough it didn't fail. My original thought was that Cleanup should execute even if job fails to generate configuration maps, but I see in that case this method would surely fail.
I restored older Cleanup and added job build directory removal in finally brackets after Cleanup.